exercises/practice/resistor-color-duo/.meta/example.ex

```diff
@@ -0,0 +1,22 @@
+defmodule ResistorColorDuo do
+  @colors %{
+    black: 0,
+    brown: 1,
+    red: 2,
+    orange: 3,
+    yellow: 4,
+    green: 5,
+    blue: 6,
+    violet: 7,
+    grey: 8,
+    white: 9
+  }
+
+  @doc """
+  Calculate a resistance value from two colors
+  """
+  @spec value(colors :: [atom]) :: integer
```

Comment:
I like that the colors are atoms. What do you think about editing `resistor-color` to also use atoms for the colors? Now they're strings.

Reply:
Leave it to me. Maybe we should make the trio all a little bit more consistent on other things as well?

Reply:
🤷🏻 [6 of one, half dozen of the other](https://en.wikipedia.org/wiki/Six_of_One#:~:text=The%20idiom%20%22six%20of%20one,matter%20which%20one%20we%20choose.)